### PR TITLE
swagger docs

### PR DESCRIPTION
**Pull request by Schparky**

### CODE REVIEW

File: application/actions/claims.go [hunk 1]
```diff
@@ -23,6 +37,23 @@ func claimsList(c buffalo.Context) error {
...
 
+// swagger:operation GET /claims/{id} Claims ClaimsView
+//
+// ClaimsView
+//
+// view a specific claim
+//
+// ---
+// parameters:
+// - name: id
```
Comment (Baggerone) [inline application/actions/claims.go:48]: At one point, we thought we didn't need to have a note about these obvious path parameters. 
Do you prefer having them?

Comment (Schparky) [reply application/actions/claims.go:48]: I don't recall, but it seemed to be necessary for the generated docs

Comment (Schparky) [reply application/actions/claims.go:48]: Also, since they're all `id` it seems to add a bit of clarity

File: application/actions/items.go [hunk 2]
```diff
@@ -56,6 +99,29 @@ func itemsCreate(c buffalo.Context) error {
 	return c.Render(http.StatusOK, r.JSON(output))
 }
 
+// swagger:operation PUT /policies/{id}/items PolicyItems PolicyItemsUpdate
```
Comment (Baggerone) [inline application/actions/items.go:102]:
```suggestion
// swagger:operation PUT /items/{id} PolicyItems PolicyItemsUpdate
```

File: application/actions/claims.go (hunk 1 above)
Comment (fillup) [reply application/actions/claims.go:48]: is there any annotation for these to indicate that it should be a uuid v4?

**Review by Baggerone — APPROVED** [review]

File: application/actions/items.go (hunk 2 above)
Comment (fillup) [inline application/actions/items.go:102]: This should probably be `PUT /items/{id}` if we already have the item ID, and the update model might need to be different than a create. We need to think through what we'd allow to be edited over time as some properties may need to trigger new workflows, like changing the item value to something above an auto-approval amount would need to go through approval, etc.

File: application/api/items.go [hunk 3]
```diff
@@ -21,49 +26,103 @@ type Items []Item
...
+	// model
+	Model string `json:"model"`
+
+	// serial number
+	SerialNumber string `json:"serial_number"`
+
+	// coverage amount (USD)
+	CoverageAmount int `json:"coverage_amount"`
 
+	// date (yyyy-mm-dd) of item's purchase
```
Comment (fillup) [inline application/api/items.go:67]: any idea if this short date format same as goswagger's `swagger:strfmt date` format? the docs don't provide an example that I could see. https://goswagger.io/use/spec/strfmt.html

**Review by fillup — APPROVED** [review]

File: application/api/items.go (hunk 3 above)
Comment (Schparky) [reply application/api/items.go:67]: image

File: application/actions/items.go (hunk 2 above)
Comment (Schparky) [reply application/actions/items.go]: thanks

File: application/actions/claims.go (hunk 1 above)
Comment (Schparky) [reply application/actions/claims.go:48]: nope

**Pull request merged**